modules/layers/src/path-layer/path-layer-vertex.glsl.js

```diff
@@ -58,18 +60,18 @@ float flipIfTrue(bool flag) {
 vec3 lineJoin(
   vec3 prevPoint, vec3 currPoint, vec3 nextPoint,
   float relativePosition, bool isEnd, bool isJoint,
-  float width
+  vec2 width
 ) {
-  vec2 deltaA = currPoint.xy - prevPoint.xy;
-  vec2 deltaB = nextPoint.xy - currPoint.xy;
+  vec2 deltaA = (currPoint.xy - prevPoint.xy) / width;
+  vec2 deltaB = (nextPoint.xy - currPoint.xy) / width;
 
   float lenA = length(deltaA);
   float lenB = length(deltaB);
```

Comment:
Perf note unrelated to this PR: It seems like this logic mostly compares length and doesn't really care about the value, so you can get a bit of a win by avoiding the square root until it's necessary. I.e.:
```cpp
float lenA2 = dot(deltaA, deltaA);
float lenB2 = dot(deltaB, deltaB);
lenA2 = lenA2 > EPSILON2 ? lenA2 : 0.0;
lenB2 = lenB2 > EPSILON2 ? lenB2 : 0.0;

// Later...
if (cornerPosition < 0.0) {
  float L = sqrt(L2);
  offsetScale = min(offsetScale, L / max(cosHalfA, EPSILON));
}
```

Reply:
Will do in a follow up PR.